packages/dds/merge-tree/src/properties.ts

```diff
@@ -87,7 +88,8 @@ export function clone<T>(extension: MapLike<T> | undefined) {
 	// eslint-disable-next-line guard-for-in, no-restricted-syntax
 	for (const key in extension) {
 		const v = extension[key];
-		if (v !== null) {
+		// TODO Non null asserting, why is this not null?
+		if (v !== null && v !== undefined) {
```

Comment:
i don't like including behavioral changes as part of this PR except where they are statically analyzable. Changing the behavior of clone/extends here might have unexpected consequences in other bits of merge tree.